README.md

```diff
@@ -1,75 +1,126 @@
+TensorHive
+===
 
-Core feature of TensorHive is to allow for easy execution of distributed trainings without the need for manual logging into every worker node or installing and configuring resource management daemons.
 
-Further plans include: 
-* task and hardware queuing
-* support for machine learning applications in frameworks other than TensorFlow
+![](https://img.shields.io/badge/release-v0.2-brightgreen.svg?style=popout-square)
+![](https://img.shields.io/badge/pypi-v0.2-brightgreen.svg?style=popout-square)
+![](https://img.shields.io/badge/platform-Linux-blue.svg?style=popout-square)
+![](https://img.shields.io/badge/python-3.5%20|%203.6%20|%203.7-blue.svg?style=popout-square)
+![](https://img.shields.io/badge/license-Apache%202.0-blue.svg?style=popout-square)
 
-## Roadmap:
-(Due to funding delays, the milestones have been modified)
+TensorHive is an open source system for managing and monitoring your computing resources across multiple hosts.
+It solves the most common problems and nightmares about accessing and sharing your AI-focused infrastructure across multiple, often competing users.
 
-### Resource reservation module
-  * Beta: **31.07.2018**
-  * Final release: **31.12.2018**
-  
-### Executing distributed TensorFlow applications
-  * Beta: **30.09.2018**
-  
-## Current features
-- [x] Connect to multiple nodes via SSH
-- [x] Monitor basic GPU parameters on every host
-- [x] Define very basic OpenAPI specification
-- [x] Command Line interface
-- [ ] Users can make reservations using calendar
-- [ ] Priviliged users can view resource utilization on charts (CPU, GPU, mem, disk)
-- [ ] During the reservation period users can freely access declared resources
-- [ ] When reservation time expires (or before it starts), user processes are stopped
-- [ ] Admin dashboard (ban users, edit/reject reservations)
-
-## Demo
-
-[![asciicast](https://asciinema.org/a/hzQMCvvZMqtv8mtCafQ0l4TFk.png)](https://asciinema.org/a/hzQMCvvZMqtv8mtCafQ0l4TFk)
-
-<table>
-<thead>
-<tr>
-<th>tensorhive run core</th>
-<th>tensorhive run api</th>
-</tr>
-</thead>
-<tbody>
-<tr>
-<td><img src="https://i.imgur.com/lkTv5xH.png" alt="tensorhive run core - screenshot"></td>
-<td><img src="https://i.imgur.com/sR4PAbZ.png" alt="tensorhive run api - screenshot"></td>
-</tr>
-</tbody>
-</table>
-
-## Requirements
-
-* cluster nodes should be accessible by SSH without password
-* GPUs should support utilization monitoring through ```nvidia-smi```
-
-## Installation
+It's designed with __flexibility, lightness and configuration-friendliness__ in mind. 
+
+Test bed and benchmarks
+--------
+[**NVIDIA DGX STATION (4x NVIDIA Tesla® V100 32GB)**](https://www.nvidia.com/content/dam/en-zz/Solutions/Data-Center/dgx-station/dgx-station-print-dgx-station-32GB-infographic-final-LR.pdf) and a bunch of nodes with GTX 1060 on board.
+
+You can check out our **unique** benchamark results with a full set instructions to reproduce (both in Distributed TensorFlow):
```

Comment:
Mentioning the benchmarks is good, but it's too early in the READMe, giving an impression that it is some kind of a core functionality of TensorHive. Should be somewhere at the botton, maybe the whole "Test bed and benchmarks" section. Here we should provide TH screenshots.

Reply:
And BTW there is a typo: "benchamark"